posts/2023-09-29_gnomobile/README.md

```diff
@@ -70,7 +82,7 @@ If you want to add a new message to a blockchain, you need to actually interact
 Gno nodes run on GnoVMs (gnovm), and for the moment, these are only available on desktops. We believe it is possible to make them available on mobile as well, but we need to find clever ways to overcome the constraints of mobile devices (like putting the apps in the background (iOS), addressing network bandwidth limitations, and so on).
 
 1. Developing a **decentralized push notification service** for *both* mobile and desktop apps. Getting notifications is now a standard (and very important) functionality of centralized apps. Technically, this happens via a central server. Naturally, having a centralized server is not possible for a p2p app, but there are other ways to implement notifications, and we are considering including them in the GnoMobile framework.
-2. Making it possible for decentralized apps to **interact with the blockchain even if the network connection is poor or virtually unavailable**. Through the **[Wesh Network** protocol](https://wesh.network/), we are opening up the possibility of using alternative transport mediums to exchange messages between peers in an asynchronous but reliable manner in off-grid environments. Enabling reliable, secure, and censorship-resistant communication is our main cause at Berty Technologies. We want to open the door for p2p users to send messages and interact even in extreme situations or adverse scenarios, and Wesh Network is built specifically for this purpose. It is only natural to make it easier for developers to use it through the GnoMobile framework.
+2. Making it possible for decentralized apps to **interact with the blockchain even if the network connection is poor or virtually unavailable**. Through the **[Wesh Network](https://wesh.network/)** protocol, we are opening up the possibility of using alternative transport mediums to exchange messages between peers in an asynchronous but reliable manner in off-grid environments. Enabling reliable, secure, and censorship-resistant communication is our main cause at Berty Technologies. We want to open the door for p2p users to send messages and interact even in extreme situations or adverse scenarios, and Wesh Network is built specifically for this purpose. It is only natural to make it easier for developers to use it through the GnoMobile framework.
```

Comment:
```suggestion
2. Making it possible for decentralized apps to **interact with the blockchain even if the network connection is poor or virtually unavailable**. Through the [**Wesh Network** protocol](https://wesh.network/), we are opening up the possibility of using alternative transport mediums to exchange messages between peers in an asynchronous but reliable manner in off-grid environments. Enabling reliable, secure, and censorship-resistant communication is our main cause at Berty Technologies. We want to open the door for p2p users to send messages and interact even in extreme situations or adverse scenarios, and Wesh Network is built specifically for this purpose. It is only natural to make it easier for developers to use it through the GnoMobile framework.
```

Reply:
Why is encapsulating the entire hyperlink in bold incorrect? 

>I love supporting the **[EFF](https://eff.org)**.
```
I love supporting the **[EFF](https://eff.org)**.
```
I am following [this markdown guide](https://www.markdownguide.org/basic-syntax/#formatting-links). Should I be using a different guide?

Reply:
The issue is not with Markdown. Both solutions are correct. However, your fix alters the intended behavior, which is to have "Wesh Network" in bold and include the protocol in the link.

---

Before: Through the **[Wesh Network** protocol](https://wesh.network/), we are opening...

Your fix: Through the **[Wesh Network](https://wesh.network/)** protocol, we are opening...

My fix: Through the [**Wesh Network** protocol](https://wesh.network/), we are opening...

---

Reply:
Got it, thank you for the clarification